functions-and-scope.js

```diff
@@ -53,12 +85,30 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // averageGrade([6, 4, 5]) geeft xxxx
 // averageGrade([8, 9, 4, 6, 10]) geeft xxxx
 
+function averageGrade (cijferArray) {
+    let sum = 0;
+
+    for (let i = 0; i < cijferArray.length; i++) {
+        sum+= cijferArray[i];
+    }
+
+   return average = sum / cijferArray.length;
+}
+
+console.log(averageGrade(grades));
+
 
 /* 2c: Afronden op twee decimalen */
 // Zorg ervoor dat het gemiddelde cijfer dat wordt teruggegeven uit de functie netjes wordt afgerond op twee decimalen.
 // Tip: Google is your best friend!
 
 
+function rounder2Decimals(xy){
+    return Number.parseFloat(xy).toFixed(2)
```

Comment:
De parse hoeft niet perse:
```suggestion
    return xy.toFixed(2);
```